tools/codegen/pkg/interfacegen/testdata/BUILD

```diff
@@ -0,0 +1,84 @@
+package(default_visibility = ["//tools/codegen/pkg/interfacegen:__pkg__"])
+
+load("@org_pubref_rules_protobuf//protobuf:rules.bzl", "proto_compile")
+load("//tools/codegen:generate.bzl", "mixer_proto_library")
+
+mixer_proto_library(
+    name = "list_template_library",
+    testonly = True,
+    protos = ["ListTemplate.proto"],
```

Comment:
Should we add a tests that wants to import things outside the well known imports (which mixer_proto_library will include for them)?